src/config/backpack/themes/coreui2.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+return [
+    'class' => [
+        'header' => 'app-header bg-light border-0 navbar',
+        'body' => 'app aside-menu-fixed sidebar-lg-show',
+        'sidebar' => 'sidebar sidebar-pills bg-light',
+        'footer' => 'app-footer d-print-none',
+    ],
+
+    /**
+     * Styles.
+     */
+    'styles' => [
+        'https://unpkg.com/@coreui/coreui@2.1.16/dist/css/coreui.min.css',
+        'https://unpkg.com/animate.css@4.1.1/animate.compat.css',
+        'https://unpkg.com/noty@3.2.0-beta-deprecated/lib/noty.css',
+
+        ['https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,400;0,600;0,700;1,400&display=swap', true, [], 'style'],
+
+        'https://cdnjs.cloudflare.com/ajax/libs/line-awesome/1.3.0/line-awesome/css/line-awesome.min.css',
+        'https://cdnjs.cloudflare.com/ajax/libs/line-awesome/1.3.0/line-awesome/fonts/la-regular-400.woff2',
+        'https://cdnjs.cloudflare.com/ajax/libs/line-awesome/1.3.0/line-awesome/fonts/la-solid-900.woff2',
+        'https://cdnjs.cloudflare.com/ajax/libs/line-awesome/1.3.0/line-awesome/fonts/la-brands-400.woff2',
```

Comment:
This array should be empty by default. So let's move these to blade files - some in parent theme, some in child theme.